### PR TITLE
windterm@2.4.1: Fix for changed way of storing profiles by the app

### LINKED ISSUE #8409: WindTerm changed the way how the connection profiles are stored.
## Bug Report

**Package Name:** WindTerm

### Current Behaviour

Application manifest defines four persisted folders:
- profiles: connection profiles
- global: configuration of the app layout, menu structure and function, keyboard shortcuts
- plugins: plugins for Qt platform on which is app built
- terminal: configuration of terminal protocols

Beginning by the version 2.4.0, connection profiles are stored on another places, see the [release notes](https://github.com/kingToolbox/WindTerm/releases), section *Upgrade*.
Subfolder 'profiles' no more fits to the app behavior from version 2.4.0.
Subfolder 'global' is intended to be static, created by developer, not changed by users. For user's customizations there is subfolder 'global' in the profile folder.
Subfolder 'plugins' should not be changed by the user, as it is critical for the application run.
Subfolder 'terminal' is intended to be static, created by developer, not changed by users. For user's customizations there is subfolder 'terminal' in the profile folder.

### Possible Solution

Leave just one persisted folder named '.wind' and configure the app to use profile stored along the application files.

### PR DESCRIPTION
**Pull request by vozkych**
Only one persisted folder named '.wind' is defined, notes with suggestion how to configure the app to use profile stored along the application files added.

Closes #8409

- [x] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

File: bucket/windterm.json [hunk 1]
```diff
@@ -6,6 +6,12 @@
     "suggest": {
         "vcredist2022": "extras/vcredist2022"
     },
+    "notes": [
+        "From version 2.4.0, WindTerm changed the way how the connection profiles are stored.",
+        "During the first run of version 2.4.0 or higher, please, choose the first option for 'profile directory', so the profiles will be stored along the application files.",
+        "Then copy the old profiles directory, stored under 'persist/WindTerm/profiles', over the newly created folder under 'persist/WindTerm/.wind/profiles'. After that you can delete the old 'profiles' directory.",
```
Comment (rashil2000) [inline bucket/windterm.json:12]: Can you automate this process? Have a look at vscode's manifest to see how we handle data migrations: https://github.com/ScoopInstaller/Extras/blob/f318f78fc56137877a573495c6f56b805f52090f/bucket/vscode.json#L48

**Comment by vozkych** [comment]
@rashil2000  Thank you for the hint, the process of profiles migration is now automated.

File: bucket/windterm.json [hunk 2]
```diff
@@ -29,6 +23,14 @@
             "WindTerm"
         ]
     ],
+    "post_install": [
+        "info '[Portable Mode] Configuring profiles directory...'",
+        "'{ \"path\" : \".\" }' | Set-Content -Path \"$dir\\profiles.config\"",
```
Comment (rashil2000) [inline bucket/windterm.json:28]: You should also check for existence of this file

**Comment by vozkych** [comment]
The file is created by the application whenever a new version of the application is started for the first time. The file is located next to the application files. Each version of the application is located in its own directory. There is no need to test the file for existence, it cannot exist after installation unless it is part of the installation package (which is not).

File: bucket/windterm.json (hunk 2 above)
Comment (vozkych) [reply bucket/windterm.json:28]: Normally is the file created by the application whenever a new version of the application is started for the first time. The file is located next to the application files. Each version of the application is located in its own directory. There is no need to test the file for existence, it cannot exist after installation unless it is part of the installation package (which is not).

File: bucket/windterm.json [hunk 3]
```diff
@@ -23,11 +23,16 @@
...
+        "if (!(Test-Path \"$dir\\.wind\\profiles\") -and (Test-Path \"$persist_dir\\profiles\")) {",
+        "    info '[Portable Mode] Copying existing profiles...'",
+        "    Copy-Item \"$persist_dir\\profiles\" \"$dir\\.wind\" -Recurse",
+        "}"
+    ],
     "persist": [
```
Comment (rashil2000) [inline bucket/windterm.json:34]:
```suggestion
    "persist": ".wind",
```

File: bucket/windterm.json [hunk 4]
```diff
@@ -23,11 +23,16 @@
...
-        "global",
-        "plugins",
-        "profiles",
-        "terminal"
+        ".wind"
     ],
```
Comment (rashil2000) [inline bucket/windterm.json:36]:
```suggestion
```

**Pull request merged**